app/src/main/res/menu/menu_search.xml

```diff
@@ -5,7 +5,7 @@
     <item
         android:id="@+id/action_search"
         android:title="@string/menu_search_button"
-        android:icon="@drawable/ic_search_white_24dp"
+        android:icon="@drawable/ic_search_black_24dp"
         android:orderInCategory="1"
```

Comment:
Thanks for the PR. You have some irrelevant changes and this will give us a black search icon for both dark theme and light theme. We don't want this, you should have use ?attr